benchmark_bad_start:18:2: Calling benchmark.start when a previous benchmark.start has not been ended by a benchmark.end
